jaeger-core/build.gradle

```diff
@@ -86,12 +67,3 @@ jar {
         attributes('Implementation-Title': 'jaeger-core', 'Implementation-Version': project.version)
     }
 }
-
-task jaegerVersion << {
```

Comment:
I think this needs to remain in this jar. The property is read by the main tracer, it doesn't make sense for it to be in a thrift-only jar.